tests/e2e-image/Dockerfile

```diff
@@ -62,8 +76,16 @@ RUN wget https://github.com/drone/drone-cli/releases/download/v0.8.3/drone_linux
 RUN curl -sSL https://github.com/vmware/govmomi/releases/download/v0.16.0/govc_linux_amd64.gz | gzip -d > /usr/local/bin/govc && \
     chmod +x /usr/local/bin/govc
 
-RUN  wget https://launchpad.net/ubuntu/+source/wget/1.18-2ubuntu1/+build/10470166/+files/wget_1.18-2ubuntu1_amd64.deb && \
-     dpkg -i wget_1.18-2ubuntu1_amd64.deb
+RUN wget https://get.helm.sh/helm-v2.14.1-linux-386.tar.gz && tar zxvf helm-v2.14.1-linux-386.tar.gz && \
+    mv linux-386/helm /usr/local/bin/helm && \
+	helm init --client-only && \
+	helm plugin install https://github.com/chartmuseum/helm-push
```

Comment:
can we pin to a certain version of push plugin?

Reply:
helm3 plugin install https://github.com/chartmuseum/helm-push

Currently, we have both helm push and helm3 push commands.

Reply:
Do you mean pin helm to helm2? Currently, helm's verson is 2, helm3's version is 3.